.env.dist

```diff
@@ -93,6 +93,13 @@ teams_host='https://teams.dev.openconext.local'
 teams_username=spdashboard
 teams_password=secret
 
+## Invite instance
+invite_host='https://invite.dev.openconext.local'
+invite_api_username='sp_dashboard'
+invite_api_password='secret'
+invite_landing_url='https://example.org'
+invite_spdashboard_manage_id='4b0e422d-d0d0-4b9e-a521-fdd1ee5d2bad'
```

Comment:
I feel this could be a more generic configuration. I'd say lose the `invite_` prefix as this is the actual manage id for SP Dashboard.